files/filewriter_unix_test.go

```diff
@@ -1,4 +1,4 @@
-//go:build darwin || linux || netbsd || openbsd
+//go:build darwin || linux || netbsd || openbsd || freebsd || dragonfly || wasm
```

Comment:
```suggestion
//go:build darwin || linux || netbsd || openbsd || freebsd || dragonfly || js || wasip1
```

Reply:
@mar1n3r0 this suggestion is still missing, can you pls apply it?